packages/template-retail-react-app/app/utils/url.js

```diff
@@ -130,6 +130,7 @@ export const getPathWithLocale = (shortCode, opts = {}) => {
     const location = opts.location ? opts.location : window.location
     let {siteRef, localeRef} = getParamsFromPath(`${location.pathname}${location.search}`)
     let {pathname, search} = location
+    const urlTemplateLiteral = opts.urlTemplateLiteral
```

Comment:
If urlTemplateLiteral is not passed into opts, would this function still works? Passing it into opts indicates it could be optional. 🤔